### PR TITLE
Mx4PC metrics configuration (will be released on February 10)

### PR DESCRIPTION
**Pull request by zlogic**
In the upcoming Mx4PC release, planned for February ~3~ 10, we've added an option to use "native" metrics (Mendix 9.7 micrometer metrics), and retroactively called the current metrics mode "compatibility".

The new native metrics mode requires a separate dashboard and only works with Mendix 9.7 and newer versions, so it's possible that customers might want to use native metrics for new apps, and compatibility mode for apps based on Mendix 7 or 8.

There's also an option to collect additional metrics on native mode, exactly like [Mendix Cloud V4](https://docs.mendix.com/developerportal/operate/datadog-metrics#3-3-2-passing-a-configuration-to-the-metrics-agent). I couldn't find documentation how to use the _MxAgent instrumentation config_, but it's something customers typically use to keep Datadog costs under control.

We'll provide release notes a bit later, as a separate PR.

### CODE REVIEW

File: content/developerportal/deploy/private-cloud-monitor.md [hunk 1]
```diff
@@ -352,18 +379,26 @@ Mendix for Private Cloud offers a reference dashboard that looks similar to [Men
...
+{{% alert type="warning" %}}Depending on how Prometheus is [configured](https://prometheus.io/docs/prometheus/latest/configuration/configuration/)
+and which addons are installed, some labels or metrics might be missing or have a different name.
+The reference dashboards are compatible with Prometheus installed as described in [Installing Monitoring Tools](#install-grafana-loki), above.{{% /alert %}}
+
 {{% alert type="warning" %}}
 Mendix for Private Cloud uses a `m2ee-metrics` sidecar that collects metrics from the [admin port](/refguide/monitoring-mendix-runtime) and translates them into a format supported by Prometheus.
 This approach works with all Mendix versions, starting from Mendix 7.23.
 
-Mendix 9.6 introduces native [Prometheus metrics](/refguide/metrics).
+Mendix 9.6 introduces native [Prometheus metrics](/refguide/metrics).
```
Comment (MarkvanMents) [inline content/developerportal/deploy/private-cloud-monitor.md:390]: Is this the right version. We say Mx 9.7 in other places.

**Review by MarkvanMents — COMMENTED** [review]
Hi Dmitrii (@zlogic)
I've made some changes which you can see in this PR https://github.com/zlogic/docs/pull/1, or this commit https://github.com/zlogic/docs/commit/97378d98910fff995a8e5ef00db45638d499a413.

I've also asked a couple of questions and made some suggestions which you can commit if you want.

Let me know if you are happy with this and I'll approve.

Mark

**Comment by zlogic** [comment]
Hi @MarkvanMents, thank you for the review! I'll go through your comments, but today we decided to postpone this release for one week - to be released on February 10.

File: content/developerportal/deploy/private-cloud-cluster.md [hunk 2]
```diff
@@ -1289,6 +1289,64 @@ You can customize the registry imageNameTemplate in OperatorConfiguration with t
 * `{{.UnixTimestamp}}`: current UNIX timestamp with at least millisecond precision e.g. 1640615972.897.
 * `{{.Timestamp}}`: current timestamp in the following format 20211231.081224.789 for 2021-12-31 08:12:24.789.
 
+### 5.5 Customize runtime metrics{#customize-runtime-metrics}
```
Comment (MarkvanMents) [inline content/developerportal/deploy/private-cloud-cluster.md:1292]:
```suggestion
### 5.5 Customize Runtime Metrics{#customize-runtime-metrics}
```

File: content/developerportal/deploy/private-cloud-monitor.md (hunk 1 above)
Comment (zlogic) [reply content/developerportal/deploy/private-cloud-monitor.md:390]: Mendix 9.6 added experimental support for native metrics, but isn't feature-complete. Mendix 9.7 is the first version that provides all required metrics (number of logged in users, database queries and so on).

File: content/developerportal/deploy/private-cloud-cluster.md [hunk 3]
```diff
@@ -1289,6 +1289,64 @@ You can customize the registry imageNameTemplate in OperatorConfiguration with t
...
+      {
+        …
+      }
+  # …
+```
+
+You can set the following metrics configuration attributes:
+
+* `mode`: metrics mode, `native` or `compatibility`.
+* `duration`: Prometheus scrape duration interval in ISO 8601 Duration format (e.g. 'PT1M'), should be aligned with your Prometheus configuration; can be left empty, which defaults to 1 minute (matching the default Prometheus scrape interval). This attribute only works when `mode` is set to `native`.
```
[the remark this answers is not in the document]
Comment (zlogic) [reply content/developerportal/deploy/private-cloud-cluster.md:1337]: Yes, indeed, it's called *interval* in the Prometheus documentation. We're changing this - to reduce possible confusion; renaming this attribute after the release would be a lot more difficult.

**Comment by zlogic** [comment]
@MarkvanMents I've made some changes to the document to reflect the latest UI changes, can you take a look?

File: content/developerportal/deploy/private-cloud-monitor.md (hunk 1 above)
Comment (MarkvanMents) [reply content/developerportal/deploy/private-cloud-monitor.md:390]: OK, I'll leave it then.

**Review by MarkvanMents — APPROVED** [review]
Just made a couple of proofreading corrections, but the content looks great.
Thanks for giving me enough time to review it.
Let me know if it can be published tomorrow.
Mark

**Pull request merged**